feat: 图表模块开发及todotip优化

tunshiyu opened the pull request:
1 图表模块基础表格/基础流图/基础饼图开发
2 引入神威基础流图的option方法
3 优化todotip
4注释掉tabs页面的轮播图console

tunshiyu committed:
feat: 图表模块开发及todotip优化

1 图表模块基础表格/基础流图/基础饼图开发
2 引入神威基础流图的option方法
3 优化todotip
4注释掉tabs页面的轮播图console

SunshineH2 commented on src/components/ChartWrap/index.tsx:
```diff
@@ -0,0 +1,27 @@
+  padding?: number; //
+}
+const ChartWrap: React.FC<ChartWrapProps> = props => {
+  const { title, padding = 12 } = props;
```
这个padding用px包一下

tunshiyu committed:
fix: 修改px和flex direction

tunshiyu committed:
feat: 热力地图及展示下的设置及左滑删除弹窗

1 热力地图开发,从神威拿来基础地图的option方法
2 展示下的设置页面
3 左滑删除弹窗封装

tunshiyu committed:
feat: 展示下的卡片页面开发

tunshiyu committed:
fix: 解决冲突

SunshineH2 commented on src/components/CustomTag/index.tsx:
```diff
@@ -1,8 +1,65 @@
+      borderStyle: 'solid',
+      borderColor: Color.borderColor,
+      paddingVertical: 4,
+      paddingHorizontal: 4
```
用px包一下

chj-damon commented on src/components/BlockItem/index.tsx:
```diff
@@ -0,0 +1,67 @@
+  isWithIcon?: boolean;
+}
+
+const BlockItem = ({ isSelected, name, onPress, style, isWithIcon }: BlockItemInter) => {
```
改一下名字，CustomTag

chj-damon commented on src/components/CustomTag/index.tsx:
```diff
@@ -1,8 +1,65 @@
+  type?: 'blue' | 'green' | 'yellow' | 'red';
+}
+
+const CustomTag: React.FC<CustomTagProps> = ({ children, style, type = 'blue' }) => {
```
把这个组件和上面的BlockItem合并成一个组件。他们的区别只在于是否可以点击

chj-damon commented on src/components/ImgLinkItem/index.tsx:
```diff
@@ -0,0 +1,37 @@
+  navigateTo?: string; //跳转url
+}
+
+const ImgLinkItem = ({ source, title, content, navigateTo }: BlockItemInter) => {
```
什么作用？

chj-damon commented on src/screens/show/components/Card/index.tsx:
```diff
@@ -1,12 +1,105 @@
+import ImgLinkItem from '../../../../components/ImgLinkItem';
+const px = Size.px;
+
+export default function Cardd() {
```
拼写错误

chj-damon commented on src/components/SearchPicker/index.tsx:
```diff
@@ -0,0 +1,169 @@
+      maxShowLength: 9
+    }
+  };
+  if (size === 'small' || size === 'mid' || size === 'big') {
```
我觉得既然有size了，是不是就不需要额外传自定义的width和maxShowLength了？这种自定义的需求感觉很弱，开发人员只需要关心size一个属性就好。

chj-damon commented on src/components/SearchPicker/index.tsx:
```diff
@@ -0,0 +1,169 @@
+  };
+
+  // 当cols为2时，根据searchIndex进行筛选
+  let listData =
```
useMemo包一下

tunshiyu committed:
fix: comment修改

1 合并静态和动态tag 2 searchPicker只提供size配置来简化使用 3 使用px包裹 4 searchPicker搜索后的data使用memo包裹

tunshiyu committed:
fix: 优化代码
